examples/sentiment/train_sentiment.py

```diff
@@ -88,24 +88,43 @@ def read_corpus(path, vocab):
         return trees
 
 
-def traverse(node, train=True, evaluate=None, root=True):
+class RecursiveNet(chainer.DictLink):
+
+    def __init__(self, n_vocab, n_units):
+        super(RecursiveNet, self).__init__(
+            embed=F.EmbedID(n_vocab, n_units),
```

Comment:
Use chainer.links instead of chainer.functions (also in the following two lines)